### PR TITLE
875 rework miniconda docs

### PR DESCRIPTION
**Pull request by kathatherine**
### Description

Closes https://github.com/conda/conda-docs/issues/875.

This PR updates the Miniconda docs a great deal. It moves most information to its own pages, to clean up the main page, and adds some quick command line install commands (as requested in https://github.com/conda/conda-docs/issues/842).

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] ~Add / update necessary tests?~
- [ ] ~Add / update outdated documentation?~

### CODE REVIEW

File: docs/source/miniconda-install.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+           c. Check the hash that appears against the hash listed next to the installer you downloaded. See :doc:`all Miniconda installer hashes here <miniconda_hashes>`.
+        
+        #. Double-click the `.pkg` file.
+        #. Follow the instructions on the screen. If you are unsure about any setting, accept the defaults. You can change them later.
+        #. When the installation finishes, open your terminal application.
+        #. Test your installation by running ``conda list``. If conda has been installed correctly, a list of installed packages appears.
+
+        `More information on installing in silent mode on macOS is in the conda project documentation <https://conda.io/projects/conda/en/stable/user-guide/install/macos.html>`_.
+
+    .. tab-item:: Linux
```
Comment (kathatherine) [inline docs/source/miniconda-install.rst:49]: Haven't finished the Linux bit yet.

**Comment by kathatherine** [comment]
@marcoesters This is ready for you when you're ready.

**Comment by kathatherine** [comment]
I think the current pre-commit issue can be fixed with merging main once https://github.com/conda/conda-docs/pull/877 is merged in.

**Review by marcoesters — DISMISSED** [review]

**Comment by kathatherine** [comment]
pre-commit.ci autofix

**Review by jezdez — CHANGES_REQUESTED** [review]
We need to split the miniconda docs into a separate repo that we publish like the other projects in ReadTheDocs. It's uncanny how an Anaconda product (miniconda) is intermingled with the conda Open Source docs preventing us from decoupling the work on conda projects.

**Comment by kathatherine** [comment]
@jezdez Yes, I figured we'd do this work and then split it off from the conda-docs repo in a different PR. But if you want to do it differently, we should discuss that.

**Comment by marcoesters** [comment]
> We need to split the miniconda docs into a separate repo that we publish like the other projects in ReadTheDocs. It's uncanny how an Anaconda product (miniconda) is intermingled with the conda Open Source docs preventing us from decoupling the work on conda projects.

While I agree with you, I think this is out of scope for this PR.

I suggest we migrate this into a separate repo after this PR is merged.

**Comment by jezdez** [comment]
> > We need to split the miniconda docs into a separate repo that we publish like the other projects in ReadTheDocs. It's uncanny how an Anaconda product (miniconda) is intermingled with the conda Open Source docs preventing us from decoupling the work on conda projects.
> 
> While I agree with you, I think this is out of scope for this PR.
> 
> I suggest we migrate this into a separate repo after this PR is merged.

Sounds good to me, just wanted to make sure we're clear on the next steps and don't duplicate efforts.

**Review by jezdez — APPROVED** [review]

**Pull request merged**